src/wifes_calib.py

```diff
@@ -793,36 +793,55 @@ def derive_wifes_telluric(cube_fn_list,
             ex_data = numpy.loadtxt(extract_in_list[i])
             obs_wave = ex_data[:,0]
             obs_flux = ex_data[:,1]
+        
+        # Filtering nan values in case of missing flux values
+
```

Comment:
Prefer no newline after comment, and single newlines below.

Reply:
Good, fixed!